physics.py

```diff
@@ -44,11 +44,12 @@ def __init__(
         gearing: float,
         moi: kilogram_square_meters,
     ):
+        self.plant = LinearSystemId.DCMotorSystem(DCMotor.falcon500(), moi, gearing)
```

Comment:
I think you may want to create a single motor and pass it here and to the sim. Things might get a bit funny if those things somehow diverge in state